oracle/datadog_checks/oracle/oracle.py

```diff
@@ -87,7 +87,10 @@ def _get_config(self, instance):
         service = instance.get('service_name', None)
         jdbc_driver = instance.get('jdbc_driver_path', None)
         tags = instance.get('tags', [])
-        custom_queries = instance.get('custom_queries', [])
+        custom_queries = []
+        if self.init_config is not None:
+            custom_queries = self.init_config.get('custom_queries', [])
```

Comment:
I think we should ultimately dedupe (if there is the same query in both instances + init_config, we shouldn't query both. It would be a configuration error, but we should handle that case)

Additionally, I think we should add a flag, something like `use_global_query` and have it default to True at the instance level. If it's set to True, then that instance will append the queries from init_config, if it's set to False, then it will ONLY query what was defined at the instance level. 

This would cover the use case where a user would want a query across most of their instances, but perhaps doesn't apply to a staging env. They could then set `use_global` to false in the staging instance and define their own queries.

Reply:
I think we should just override what's in `init_config` with what you define in an instance, and make it clear in the example conf that's how it's done. That's already what we do for other checks where it's possible to define instance parameters in the init_config.

Reply:
Do you have an example of another check? It feels like the right behavior here is to not override. You could have a few common queries and perhaps a query specific to prod/staging or something.

Reply:
Actually I have no example of config parameters that are arrays like this 🙈. I was thinking of simple parameters.
I can see the value of not always overriding. Your solution with an additional parameter might be the best option.

Reply:
I imagine some hash method would be needed, maybe on the columns being selected and the table being queried. May not be super straightforward, but I think would be useful.

Reply:
I like this idea